.github/workflows/release.yaml

```diff
@@ -1,20 +1,18 @@
 on:
   push:
     branches:
-      - master
+      - remove-release-action-2
 name: release-please
 jobs:
   release-please-pr:
     runs-on: ubuntu-latest
     steps:
       - id: release-pr
-        uses: GoogleCloudPlatform/release-please-action@v2
+        uses: google-github-actions/release-please-action@v2
         with:
-          token: ${{ secrets.FORKING_TOKEN }}
-          release-type: node
           fork: true
-          package-name: googleapis
-          command: release-pr
+          token: ${{ secrets.FORKING_TOKEN }}
+          command: manifest
```

Comment:
I'm sure you're aware but the way I wrote this it's gonna execute both `manifest-pr` and `manifest-release` (whereas before you just had `release-pr` and not `github-release`)

Reply:
For this repo, we want `manifest-pr` only (good catch).